_positions/visual-designer.md

```diff
@@ -1,40 +1,29 @@
 ---
 title: Visual Designer
-permalink: who-we-are-hiring/positions/visual-designer/
+permalink: roles-and-teams/positions/visual-designer/
 team: design
 active: false
```

Comment:
Checking: should this be `false`? If you want this position to appear eventually but not yet, I suggest making a placeholder issue for changing the relevant listings' values from `false` to `true` at the appropriate time.